src/autocomplete/autocomplete-types.ts

```diff
@@ -45,6 +45,26 @@ export interface ClickHouseAutocompleteResult extends AutocompleteResultBase {
     suggestEngines?: EngineSuggestion;
 }
 
+type YQLEntity =
+    | 'externalDataSource'
+    | 'view'
+    | 'object'
+    | 'tableStore'
+    | 'table'
+    | 'replication'
+    | 'topic'
+    | 'group'
+    | 'user';
+export interface YQLAutocompleteResult
+    extends Omit<AutocompleteResultBase, 'suggestViewsOrTables'> {
```

Comment:
`AutocompleteResultBase` should contain things that exist in each database. Please remove `suggestViewsOrTables` from base, and add it to postgresql, mysql, clickhouse separatelly. It will get rid of your `Omit`